index.html

```diff
@@ -4,20 +4,21 @@
 	<title>Monopoly</title>
 	<meta http-equiv="Content-type" content="text/html;charset=UTF-8" />
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
-	<script type="text/javascript" src="classicedition.js"></script>
+	<script type="text/javascript" src="http://isaiahpatton.github.io/monopoly/classicedition.js"></script>
 	<!-- Comment out the previous line and uncomment the next one to enable the New York City Edition -->
 	<!-- <script type="text/javascript" src="newyorkcityedition.js"></script> -->
-	<script type="text/javascript" src="ai.js"></script>
-	<script type="text/javascript" src="monopoly.js"></script>
-	<link rel="stylesheet" type="text/css" href="styles.css" />
+	<script type="text/javascript" src="//isaiahpatton.github.io/monopoly/ai.js"></script>
+	<script type="text/javascript" src="//isaiahpatton.github.io/monopoly/monopoly.js"></script>
+	<link rel="stylesheet" type="text/css" href="//isaiahpatton.github.io/monopoly/styles.css">
+	<CENTER><img src="https://raw.githubusercontent.com/IsaiahPatton/monopoly/gh-pages/images/mlogo.png"></CENTER>
```

Comment:
Relative paths should be used for all scripts and styles, with the exception of specific CDNs like with jquery.